cmd/node-joiner/testdata/add-nodes.txt

```diff
@@ -0,0 +1,134 @@
+# Verify that the add-nodes command generates correctly the ISO image
+
+exec node-joiner add-nodes --kubeconfig=$WORK/kubeconfig --log-level=debug --dir=$WORK
+exists $WORK/agent-addnodes.x86_64.iso
+
```

Comment:
Should we also assert the contents of agent-addnodes.x86_64.iso contain the correct values in /etc/assisted/manifests to add the extra-worker-0 node to the cluster?